services/web/server/src/simcore_service_webserver/login/handlers_registration.py

```diff
@@ -130,7 +130,10 @@ async def register(request: web.Request):
     )
 
     # NOTE: PC->SAN: should this go here or when user is actually logged in?
-    await auto_add_user_to_groups(request.app, user["id"])
+    await auto_add_user_to_groups(app=request.app, user_id=user["id"])
+    await auto_add_user_to_product_group(
```

Comment:
what about the NOTE here? didn't you want to change it?

Reply:
Will do in a separate PR:
- https://github.com/ITISFoundation/osparc-simcore/issues/3709